web/routes/web.php

```diff
@@ -149,6 +149,12 @@
 Route::view('--/privacy', 'next.test.page-test')->name('privacy');
 Route::view('--/report-flag', 'next.test.page-test')->name('report-flag');
 
+// Dashboard
+Route::view('/--/dashboard/{path?}', 'next.wiki.dashboard')
```

Comment:
This starts with `/--` instead of `--` (compared to the others), is this intentional?

Reply:
lol no, I don't think it changes anything. I'll change it to match the others